### PR TITLE
Quadtree

### PR DESCRIPTION
**Pull request by timaktimak**

### CODE REVIEW

File: QuadTree/QuadTree.playground/Sources/QuadTree.swift [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+  }
+}
+
+extension Rect: CustomStringConvertible {
+  public var description: String {
+    return "Rect(\(origin), \(size))"
+  }
+}
+
+protocol PointsContainter {
```
Comment (kelvinlauKL) [inline QuadTree/QuadTree.playground/Sources/QuadTree.swift]: Do you mean `PointsContainer`?

File: QuadTree/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# QuadTree
+
+A quadtree is a [tree](https://github.com/raywenderlich/swift-algorithm-club/tree/master/Tree) in which each internal (not leaf) node has four children.
```
Comment (kelvinlauKL) [inline QuadTree/README.md]: A diagram here to start things off would be extremely valuable.

**Comment by timaktimak** [comment]
It was definitely a typo, sorry about that! Thanks!

**Comment by kelvinlauKL** [comment]
This looks great. Thanks @timaktimak  👍

**Pull request merged**